functions_and_scope_lab.js

```diff
@@ -0,0 +1,286 @@
+// # Function Exercises
```

Comment:
Good job overall for this assignment. No major feedback or errors. Keep up the good work and try to solve problem 10 on your free time.